packages/p2p/src/components/buy-sell/buy-table.jsx

```diff
@@ -46,17 +47,22 @@ export class BuyTable extends React.Component {
         if (api_error_message) return <TableError message={api_error_message} />;
 
         return items.length ? (
-            <InfiniteLoaderList
-                // screen size - header size - footer size - page overlay header - page overlay content padding -
-                // tabs height - padding+margin of tab content - toggle height - table header height
-                initial_height={'calc(100vh - 48px - 36px - 41px - 2.4rem - 36px - 3.2rem - 40px - 52px)'}
-                items={items}
-                RenderComponent={Row}
-                RowLoader={BuySellRowLoader}
-            />
+            <BuySellTable>
+                <InfiniteLoaderList
+                    // screen size - header size - footer size - page overlay header - page overlay content padding -
+                    // tabs height - padding+margin of tab content - toggle height - table header height
+                    initial_height={'calc(100vh - 48px - 36px - 41px - 2.4rem - 36px - 3.2rem - 40px - 52px)'}
+                    items={items}
+                    RenderComponent={Row}
+                    RowLoader={BuySellRowLoader}
+                />
+            </BuySellTable>
         ) : (
             <div className='deriv-p2p__empty'>
-                {localize("No ads yet. If someone posts an ad, you'll see it here.")}
+                <div className='deriv-p2p__empty-item'>
+                    <Icon icon='IcNoAd' size={128} />
+                    <div className='deriv-p2p__empty-text'>{localize('No ads found')}</div>
```

Comment:
we were using this text in the past and changed it after discussion with CW because it sounds like we hit a server error and couldn't find ads. why are we changing it back to this again?

Reply:
Ahh, I didn't knew that. I'll double check with them. May i know which member of CW should I confirm with?

Reply:
I have confirm with CW team. We will used what we see in design.